crates/astria-sequencer/src/transaction/action_handler.rs

```diff
@@ -5,9 +5,12 @@ use penumbra_storage::{
     StateWrite,
 };
 
+use crate::accounts::types::Address;
+
 #[async_trait]
 pub(crate) trait ActionHandler {
     fn check_stateless(&self) -> Result<()>;
```

Comment:
If a lot (most?) actions don't do anything here anyways, you could provide a default impl for this:
```suggestion
    fn check_stateless(&self) -> Result<()> {
        Ok(())
    }
```
Then you can only impl that where you actually want custom behavior.